net/rhp/contracts.go

```diff
@@ -63,18 +63,6 @@ func PaymentRevision(fc types.FileContract, amount types.Currency) (types.FileCo
 	return fc, nil
 }
 
-// ClearingRevision returns a new file contract with the revision
-// number set to uint64 max, the file fields set to their zero value, and the
-// missed proof outputs set to the valid proof outputs.
-func ClearingRevision(fc types.FileContract) types.FileContract {
```

Comment:
we could keep this around as:
```go
func FinalizationRevision(fc types.FileContract) types.FileContract {
    fc.RevisionNumber = types.MaxRevisionNumber
    return fc
}
```
but it's so trivial that I think we should just inline it where needed